building/upload-debs/rebuild.sh

```diff
@@ -1,7 +1,25 @@
 #!/bin/bash
 set -e -u
 
+if [ -z "${APT_REPOSITORY:-}" ]
+then
+	echo "Need to specify apt repository:"
+	echo "\$ export APT_REPOSITORY=<username>/<branch>"
+	exit 1
+fi
+
+if [ -z "${APT_SIGNING_KEY:-}" ]
+then
+	echo "Need to specify apt signing key:"
+	echo "\$ export APT_SIGNING_KEY=<key identifier>"
+	exit 1
+fi
+
+sed "s;\\\${APT_SIGNING_KEY};${APT_SIGNING_KEY};g" < conf/distributions.in > conf/distributions
+
 cd "$(dirname "$0")"
-reprepro -Vb . includedeb homeworld ../build-debs/binaries/homeworld-*.deb
-rsync -av --progress --delete-delay ./dists ./pool /mit/hyades/debian/
+gpg --import default-repo-signing-key.gpg
+reprepro -Vb . update
+reprepro -Vb . includedeb homeworld ../build-debs/binaries/${APT_REPOSITORY}/homeworld-*.deb
+rsync -av --progress --delete-delay ./dists ./pool /mit/hyades/apt/${APT_REPOSITORY}/
```

Comment:
You should make sure that you properly quote any substitutions like this.

Reply:
My mistake, will fix.